### ISSUE TITLE
Routing problem

### ISSUE THREAD

**Issue by Zeratuli0** [ValveSoftware/Dota2-Gameplay#19135]
### Description

OK just see how the game connect me to eu server ty valve 

### Screenshots

![20240605170103_1](https://github.com/ValveSoftware/Dota2-Gameplay/assets/166049815/3b1b43fb-1f4a-4790-b7e9-265697125a1d)

**Comment by shadowfiendult** [comment]
yep this is my main issue play in eu west but some time routed me to dubai or .... make my ping shit

**Comment by skipperarman** [comment]
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/27363135/c89a959a-a70f-4160-add9-5f75b2f62fb8)
same
